docs/install_occlum_packages.md

```diff
@@ -0,0 +1,99 @@
+# Install Occlum Packages with Package Mangement Tool
+
+Occlum packages can be easily installed and deployed using mainstream package management tool including RPM and APT.
+
+
+### Prerequisite
+
+**Install enable_RDFSBASE Kernel Module**
+
+Please follow [this README](https://github.com/occlum/enable_rdfsbase/blob/master/README.md) to install `enable_rdfsbase` kernel module.
+
```

Comment:
Need to mention the kernel module is not needed for the latest version of Linux kernel (e.g., 5.9)

Reply:
Done.